src/Selenium2Library/keywords/_element.py

```diff
@@ -297,6 +297,16 @@ def get_horizontal_position(self, locator):
             raise AssertionError("Could not determine position for '%s'" % (locator))
         return element.location['x']
 
+    def get_element_size(self, locator):
+        """Returns width and height of element identified by `locator`.
+
+        The element width and height is returned.
+        Fails if a matching element is not found.
+        """
+        element = self._element_find(locator, True, True)
```

Comment:
Passing `True` or `False` as a positional argument is always a bit confusing. Would be better to name the arguments like `something=True'.